test/simulator/quantestpy_circuit/test_add_gate.py

```diff
@@ -240,113 +243,18 @@ def test_target_qubit_empty(self,):
                 )
             )
 
-        except QuantestPyTestCircuitError as e:
+        except QuantestPyCircuitError as e:
             expected_error_msg = \
-                "quantestpy.exceptions.QuantestPyTestCircuitError: " \
+                "quantestpy.simulator.exceptions.QuantestPyCircuitError: " \
                 + "'target_qubit' must not an empty list.\n"
 
             actual_error_msg = traceback.format_exception_only(type(e), e)[0]
 
             self.assertEqual(expected_error_msg, actual_error_msg)
 
-    def test_parameter_without_param(self,):
```

Comment:
The following 4 tests are moved into `test/simulator/state_vector_circuit/test_add_gate.py`